cmd/kubeadm/.import-restrictions

```diff
@@ -26,9 +38,7 @@
 		},
 		{
```

Comment:
I'm not quite sure why import-boss even has forbidden prefixes. Based on the output I'm getting when I run hack/verify-import-boss.sh, leaving that blank is equivalent to forbidding the prefix, aside from getting a different error message ("did not match any allowed prefix" vs "has forbidden prefix"). I left it in there since nothing is needed from test.

Reply:
so there are packages like say the openstack / gophercloud do not end up being pulled (not directly but indirectly). we need forbidden prefixes to ensure that

Reply:
So should I remove the blank section or does it prevent packages getting pulled directly?